lib/ContentHelpers.php

```diff
@@ -216,13 +203,12 @@ public static function extractFlexibleContent(Entry $entry, $locale, $children =
                         }, $block->facts->all());
                     }
                     $data['content'] = $factRiver;
-                    array_push($parts, $data);
                     break;
                 case 'relatedContent':
                     $relatedContent = array();
                     $data = [
-                        'type' => $block->type->handle,
-                        'heading' => $block->heading
+                        // @TODO remove `heading` when its usage is removed from the templates
```

Comment:
Worth dropping this in as an issue? Perhaps something @mo-durvesh could take a look at?

Reply:
Good call – done: https://github.com/biglotteryfund/craft-dev/issues/246